openkat/settings.py

```diff
@@ -205,10 +205,11 @@ class SaferExceptionReporterFilter(SafeExceptionReporterFilter):
 
 ROOT_URLCONF = "openkat.urls"
 
+REDIS_PASSWORD = env.str("REDIS_PASSWORD", default=None)
 CACHES = {
     "default": {
         "BACKEND": "django_redis.cache.RedisCache",
-        "LOCATION": f"redis://:{env.str('REDIS_PASSWORD')}@{env.str('REDIS_HOST')}:6379/1",
+        "LOCATION": f"redis://:{REDIS_PASSWORD}@{env.str('REDIS_HOST', 'localhost')}:6379/1",
```

Comment:
Nice catch. Perhaps for consistency `REDIS_HOST` next to `REDIS_PASSWORD`?

```suggestion
REDIS_HOST = env.str("REDIS_HOST", "localhost")
REDIS_PASSWORD = env.str("REDIS_PASSWORD", None)
CACHES = {
"default": {
"BACKEND": "django_redis.cache.RedisCache",
        "LOCATION": f"redis://:{REDIS_PASSWORD}@{REDIS_HOST}:6379/1",
```